### DISCUSSION TITLE
Gray screen between moving gratings

### DISCUSSION THREAD

**Discussion by orena1** [bonsai-rx/bonsai#865] [Q&A] [answered]
Hello,
I have a very simple script from [bonvision](https://github.com/bonvision/) examples, that plays moving grating at different orientations from a csv file.
![image](https://user-images.githubusercontent.com/8983713/161828392-2b340e66-f709-43b0-a863-7a3d5e3e9d7e.png)

There is a 1.5sec delay between the gratings in which the screen should be gray, and it is indeed gray as long as I don't use MeshMapping. If I use MeshMapping the screen is stuck on the orientation, I've made a small video that shows the problem:
https://youtu.be/LjHYSWdrDFE

I've attached the workflow and the needed files:
[csvGratings.xls](https://github.com/bonsai-rx/bonsai/files/8421193/csvGratings.xls)
[MeshMapping_MatlabOutput.csv](https://github.com/bonsai-rx/bonsai/files/8421205/MeshMapping_MatlabOutput.csv)
[video_end_example.bonsai.zip](https://github.com/bonsai-rx/bonsai/files/8421203/video_end_example.bonsai.zip)

Thank you!
Oren

**Comment by bruno-f-cruz** [comment, marked as answer]
Hey Oren,

I am trying to reproduce this behavior on my computer I can't seem to be able to do it. I am not sure what I am doing wrong... is there a chance you attached the wrong workflow perhaps?

Cheers,

B

**Comment by orena1** [comment]
It is the correct one, after your comment I tried a different computer and it works correctly, not sure what is the problem, the computer in the lab is not very old - 9010 dell desktop.

Anyway, I've found a workaround, I'm making sure to present a zero size grating in between trials.

Thanks
